packages/zudoku/src/lib/components/context/ZudokuContext.ts

```diff
@@ -67,8 +67,7 @@ export const useCurrentNavigation = () => {
 
   const { data } = useSuspenseQuery({
     queryFn: () => getPluginSidebar(location.pathname),
-    // We just want to suspend here and don't store in SSR dehydrated state
-    queryKey: ["plugin-sidebar", NO_DEHYDRATE, location.pathname],
+    queryKey: ["plugin-sidebar", location.pathname],
```

Comment:
Unfortunately this triggered suspense boundaries to trigger after page reload. This will increase the size of the built static file but at least won't trigger a jumping page